docs/source/security/authentication.mdx

```diff
@@ -69,6 +75,9 @@ We should use this method only on highly restrictive environments that provide n
 To perform this kind of authorization, we can modify the context function:
 
 ```js
```

Comment:
Switch to `ts`?

The `// set up server, etc` here is cutting out some code that's necessary for this to have valid syntax. Right now the line below (`context: () => ...`) is just floating, and either needs to be written as a function definition:
```ts
async function context(...) {
  //...
}
```
 or as a property on an object (like how we'd normally pass it into AS)

```ts
const { url } = await startStandaloneServer(server, {
  context: async (...) => { ... },
});
```

Reply:
I hadn't considered that— I'll update to use the full `startStandaloneServer` snippet; I feel like it is easier to understand in this scenario. Thank you!